learn/get-started/get-started.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Get Started
+---
+
+The best way to start your journey with Stellar is to create a [test network](./test-net.html) account and send your first transaction.
+
+In this tutorial we will be using our [JavaScript SDK](https://www.stellar.org/developers/js-stellar-sdk/learn/index.html) but you can use other [SDKs](https://www.stellar.org/developers/horizon/learn/#libraries) as well.
+
+## Installing `stellar-sdk`
+
+First create your NodeJS project if `package.json` file is not present in your working directory:
+```
+npm init
+```
+
+Then install our JavaScript SDK:
+```
+npm install --save stellar-sdk
+```
+
+## Creating a test network account
```

Comment:
This article contains one code block per section: the first is really 6 lines long, the second is longer but it's really hard to split it (because of how promise structure looks like). I tried to explain it very well below this code block.